moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -959,6 +968,62 @@ void ompl_interface::ModelBasedPlanningContext::unregisterTerminationCondition()
   ptc_ = nullptr;
 }
 
+void ompl_interface::ModelBasedPlanningContext::omplPlannerStatusToMoveItErrorCode(
+    og::SimpleSetupPtr ompl_simple_setup, moveit_msgs::msg::MoveItErrorCodes& res)
+{
+  ompl::base::PlannerStatus ompl_status = ompl_simple_setup->getLastPlannerStatus();
+  if (ompl_status == ompl::base::PlannerStatus::UNKNOWN)
```

Comment:
Nit: Wouldn't a switch case statement work here as well? I think it can be a bit more efficient